src/main/scala/io/qbeast/spark/delta/writer/BlockWriter.scala

```diff
@@ -169,7 +169,7 @@ case class BlockWriter(
       .beginBlock()
       .setCubeId(cubeId)
       .setMaxWeight(maxWeight)
-      .setReplicated(state == State.ANNOUNCED || state == State.REPLICATED)
+      .setReplicated(state == CubeState.ANNOUNCED)
```

Comment:
Maybe we can change the `setReplicated` method to `setAnnounced`

Reply:
I think that all the Replicated/announced parts need a full review. I prefer to leave this to a new PR, as it is widespread in many places.

Reply:
Fair enough, would open a ticket 👍